hack/jenkins/e2e-runner.sh

```diff
@@ -257,12 +257,19 @@ case "${KUBERNETES_PROVIDER}" in
             cp /var/lib/jenkins/gce_keys/google_compute_engine ${WORKSPACE}/.ssh/
             cp /var/lib/jenkins/gce_keys/google_compute_engine.pub ${WORKSPACE}/.ssh/
         fi
-        if [[ ! -f ${WORKSPACE}/.ssh/google_compute_engine ]]; then
-            echo "google_compute_engine ssh key missing!"
+        echo 'Checking existence of private ssh key'
+        local _key="${WORKSPACE}/.ssh/google_compute_engine"
```

Comment:
nit: I don't think we use underscores to prefix variables anywhere else, so I'd drop it for consistency.